labcontrol/db/configuration_manager.py

```diff
@@ -14,7 +14,7 @@
 
 
 class ConfigurationManager(object):
-    """Holds the labman configuration
+    """Holds the labcontrol configuration
 
```

Comment:
Would this be "LabControl", or is there a reason for no?

Reply:
Sure!